spec/workers/enqueue_subject_queue_worker_spec.rb

```diff
@@ -53,19 +60,42 @@
       end
     end
 
-    describe "#load_subjects" do
+    context "when selecting via cellect selection strategy" do
+      it 'should attempt to queue the selected set' do
+        allow_any_instance_of(Subjects::CellectClient).to receive(:get_subjects).and_return([1])
+        expect(SubjectQueue).to receive(:enqueue)
+        subject.perform(workflow.id)
+      end
+
+      context "when the cellect client can't reach a server" do
+
+        it "should fall back to postgres strategy" do
+          allow(Subjects::CellectClient).to receive(:get_subjects)
+            .and_raise(Subjects::CellectClient::ConnectionError)
+          expect_any_instance_of(Subjects::PostgresqlSelection).to receive(:select)
+          subject.perform(workflow.id, nil, nil, nil, :cellect)
+        end
+      end
+    end
+
+    context "when subjects are selected" do
+      before do
+        allow_any_instance_of(Subjects::PostgresqlSelection).to receive(:select).and_return(result_ids)
+      end
+
       context "when there are selected subjects to queue" do
-        it 'should attempt to queue an empty set' do
-          allow_any_instance_of(PostgresqlSelection).to receive(:select).and_return([1])
+        let(:result_ids) { [1] }
+
+        it 'should attempt to queue the selected set' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.